src/internal/components/button-trigger/styles.scss

```diff
@@ -48,6 +49,11 @@ $padding-block-inner-filtering-token: 0px;
     }
   }
 
+  &.in-filtering-token-nested {
```

Comment:
This is what adds the 4px border radius to the operator, right? In case this hits some pushback in dev discussions, I'd recommend taking this part out since we want to add this to the current release, regardless of the `fixedOperations` feature.

Reply:
Sure, I will create a separate PR with this change tomorrow regardless